community/bitfetch/build

```diff
@@ -0,0 +1,4 @@
+#!/bin/sh -e
+
+make ID=kiss bitfetch-build
+make DESTDIR="$1" PREFIX="/usr" install
```

Comment:
Quotes unneeded for `/usr`.

Reply:
I know, but I'm just used to writing like that.

Reply:
Sure. It's a requirement that all build files in the "official" repositories follow the same style (which is why I request changes like this).

Reply:
Oh, ok.
I'll change that.